CODEOWNERS

```diff
@@ -0,0 +1 @@
+* @navikt/arbeid
```

Comment:
Dette burde gjøres i en annen brukerhistorie.
Uavhengig av dette så burde ikke @navikt/arbeid være codeowner siden dette er en felleskomponent som blir brukt i mange andre apper.

Reply:
Er jeg enig i at moduler skulle hatt en form for eierskap som går på tvers av produktteam. Dersom et slikt team finnes legger jeg gjerne inn det i stedet for oss.
Er også enig i at den for ryddighetens skyld kan legges inn som egen story.

Reply:
Så vidt jeg vet så er det ikke et slikt team enda. 
Man kan jo alltids ha flere codeowners, kan hende at det er det som er best inntil videre.

Reply:
Hvilke team foreslår du?

Reply:
Har laget et team nå for PTO, så du kan legge til @navikt/pto i tillegg til @navikt/arbeid